charts/dapr/charts/dapr_scheduler/templates/dapr_scheduler_statefulset.yaml

```diff
@@ -101,7 +101,7 @@ spec:
           {{- toYaml . | nindent 10 }}
         {{- end }}
         ports:
-          - containerPort: {{ .Values.ports.apiPort }}
+          - containerPort: 50006
```

Comment:
Why not making this a param, like in placement?

Reply:
Dapr would connect to scheduler on the wrong port as it's hardcoded- this is also true for placement. There isn't a reason a user would want to change this value.